data/products.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "id": "1",
+    "title": "Compton T-Shirt",
+    "description": "Sint occaecat deserunt ex incididunt qui sint. Id minim magna labore pariatur nulla quis.",
+    "price": 15,
+    "brand": "Compton",
+    "category": "T-shirt",
+    "image": "images/yellowT.png",
+    "colors": ["Red", "Green", "Blue"],
+    "sizes": ["Small", "Medium", "Large"]
+  },
+  {
+    "id": "2",
+    "title": "Comverges T-Shirt",
+    "description": "Sint occaecat deserunt ex incididunt qui sint. Id minim magna labore pariatur nulla quis.",
+    "price": 20,
```

Comment:
There should probably be some prices that have decimal points to help make this data feel more realistic.